ansible_rulebook/util.py

```diff
@@ -81,11 +81,12 @@ def substitute_variables(
         return value
 
 
-def load_inventory(inventory_file: str) -> Any:
-
-    with open(inventory_file) as f:
-        inventory_data = f.read()
-    return inventory_data
+def load_inventory(inventory_spec: str) -> Any:
+    if os.path.exists(inventory_spec):
+        with open(inventory_spec) as f:
+            inventory_data = f.read()
+        return inventory_data
+    return inventory_spec
```

Comment:
Why are you returning the string if the path does not exist? I believe that could generate errors because later the code expects to receive the content of the inventory file, like for example in https://github.com/ansible/ansible-rulebook/blob/aa7dceffd909886196f7bf122a2da20d76eabfc9/ansible_rulebook/util.py#L91

Reply:
Yeah this PR is mostly just a discussion point... I'm not sure why we *need* to parse inventory like this rather than just passing things along in a more idiomatic way to Runner underneath.

This particular PR at least allow ad-hoc inventory to be passed in. There's a ton of logic in Runner and Ansible itself to manage inventory that I don't think ansible-rulebook needs to do much if anything to it.

Reply:
@matburt Based on the [inventory notes ](https://ansible.readthedocs.io/projects/runner/en/stable/intro/#inventory)for the runner should we also allow for an env_var ANSIBLE_INVENTORY to satisfy the --inventory requirement in ansible-rulebook if the rulebook has run_playbook action? That could be the default value for [--inventory](https://github.com/ansible/ansible-rulebook/blob/67c54134be6d2bb95005463d4b1eec37b5be6c91/ansible_rulebook/cli.py#L89). It might also help if we can check the value of --inventory to see if its a file/directory/string and then do the needful based on if its 

- file that exists get copied to the Ansible-runner inventory project directory
- directory that exists gets copied along with sub directories to the Ansible-runner inventory 
- string data gets copied to inventory/hosts file like it does today